RELEASE_NOTES.md

```diff
@@ -13,6 +13,11 @@ Search Service MKII release notes
 * In KBaseSearchEngine.spec, moved the flags addNarrativeInfo and addAccessGroupInfo
```

Comment:
Not sure I follow. The comment references an unrelated release note.

Reply:
I just stuck this comment at the top of the PR diff since to have a comment as part of a review it needs to be anchored somewhere, AFAICT. The context isn't relevant.

Anyway, the comment refers to this:

> I still think that units that if we're making large changes to IndexerWorker, the steps should be
>
> 1. Add unit tests that cover the parts we're going to change
> 2. Make the changes and modify any unit tests that are invalid
>
> That ensures the changes don't break anything else. Also, we should be adding missing unit tests as we go anyway.